lightwood/config/config.py

```diff
@@ -2,16 +2,19 @@
 
 
 class CONFIG:
+    """General"""
     USE_CUDA = False
     if torch.cuda.device_count() > 0:
         USE_CUDA = True
 
     USE_DEVICE = None
+    USE_CACHE = True
```

Comment:
what are we caching?

Reply:
This is just a name I re-arranged in the config file, it was here before under the name `USE_CACHE`, arguably could be renamed to `CACHE_ENCODED_DATA`, since it's the toggle for caching encoded data.

Will change it.